### DISCUSSION TITLE
V1.0

### DISCUSSION THREAD

**Discussion by StealthChesnut** [StealthChesnut/HA-FoxESS-Modbus#8] [Announcements]
Initial release.

**Full Changelog**: https://github.com/StealthChesnut/HA-FoxESS-Modbus/commits/V1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/StealthChesnut/HA-FoxESS-Modbus/releases/tag/V1.0'>V1.0</a>.</em>

**Comment by williamjeccles** [comment]
Great Job!
